microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -19,25 +19,31 @@ All channels of Microsoft Edge have experimental features.  You can get the late
 <!-- ====================================================================== -->
 ## Experimental features which are turned on by default
 
-As of Microsoft Edge 94, the following experimental features are turned on by default.  This means that these features can be used right away, without you having to change any settings.  You can turn off these default experimental features, if needed.
+As of Microsoft Edge 96, the following experimental features are turned on by default.  This means that these features can be used right away, without you having to change any settings.  You can turn off these default experimental features, if needed.
 
-<!-- listed in this list in order per Settings > Experiments pane -->
+<!-- listed in order of the Settings > Experiments pane -->
 *  Source order viewer
+*  Enable back-forward cache debugging support
 *  [Emulation: Support dual screen mode](../device-mode/dual-screen-and-foldables.md) is turned on by default starting with Microsoft Edge 90.
+*  Enable experimental hide issues menu
+*  Enable CSS \<length\> authoring tool in the Styles pane <!-- 96 but not later has also: (https://goo.gle/length-feedback) -->
```

Comment:
I don't think we picked this up from upstream.

Reply:
(Just the "Enable CSS length" one)

Reply:
Not sure if there's action suggested here.  Copied comment to a PR that's about this page.  Closing this comment.